### DISCUSSION TITLE
Chiarimenti valvole termostatiche e vari dubbi

### DISCUSSION THREAD

**Discussion by Olodumdomotico** [indomus/forum#2968] [GENERALE - Discussioni generiche]
Salve , vorrei installare delle valvole termostatiche gestite da un bridge
- se tutte le stanze arrivano a temperatura ovviamente le valvole chiudono, ma chi spegne la caldaia??? 
- quindi serve un cronotermostato smart che pilota caldaia quando riceve info dalle valvole??
Grazie

**Comment by Scialla** [comment]
Hai le idee un pò confuse...

Ti dò qualche consiglio:

- il sistema di riscaldamento dovrebbe essere autonomo e non dipendente da HA e similari, questo perché se HA va down per qualsiasi motivo, il riscaldamento non funzionerà più. In più i sistemi autonomi hanno delle logiche difficilmente riproducibili in HA (vedi isteresi). Ha deve essere il valore aggiunto per integrare un sistema di riscaldamento autonomo e fargli fare delle cose in più, ad esempio gestire la chiusura delle valvole nelle stanze in cui viene aperta una porta/finestra o l'accensione spegnimento in base alla presenza di qualcuno; sei però libero di fare come credi e utilizzare valvole e relé su caldaia gestendo tutto da HA. Come sistemi autonomi ti consiglio TADO o NETATMO
- Se decidi di non usare un sistema autonomo ovviamente dovrai gestire tutto a mano mettendo sulla caldaia un relé smart a contatto pulito, ad es. uno shelly 1; dovrai poi crearti tu tutta la logica per gestire l'accensione e lo spegnimento della caldaia sulla base delle richieste delle valvole, nonché l'eventuale programmazione settimanale etc etc.
- Occhio che, se non hai una caldaia con il ricircolo, se tutte le valvole chiudono contemporaneamente e la caldaia (o meglio la pompa della caldaia) è ancora accesa e continua a "spingere" acqua che non riesce più a circolare, nella migliore delle ipotesi ti va in blocco la caldaia, ma potresti anche fare danni

**Comment by Olodumdomotico** [comment]
Grazie mille per la risposta ampia e anche con dei suggerimenti

Ma il problema in effetti (ovviamente ho idee confuse in materia) è perché ho crono termostato classico
Provo a spiegarmi con esempi:
-Se metto cronotermostato a 18 gradi e termostatica bagno a 20 potrei rischiare di non raggiungere mai temperatura desiderata perché caldaia stacca prima
-viceversa se metto per assurdo cronotermostato a 25 gradi e tutte le valvole a 18 gradi, la caldaia non si spegnerebbe mai
Sto sbagliando ragionamento?

**Reply by Scialla** [reply]
se usi Tado o Netatmo puoi impostare la priorità alle valvole e così, quando chiamano, la caldaia si attacca a prescindere da cosa sia impostato sul termostato.
Se invece usi il fai da te, usi un relé a contatto pulito e non usi un termostato

**Comment by Bergoglio** [comment]
Non sbagli nel ragionamento.

Il problema n. 3 (tutte le valvole chiuse e la caldaia che continua a pompare) oggettivamente lo puoi mitigare solo lato hardware mettendo sui termosifoni non delle valvole semplici bensì delle valvole a tre vie; ma dovresti chiamare l'idraulico e comunque avresti un lavoro molto anti estetico.

**Comment by Olodumdomotico** [comment]
> * quindi serve un cronotermostato smart che pilota caldaia quando riceve info dalle valvole??
>   Grazie

Quindi si torna al primo post, e la conclusione è che sarebbe poco intelligente montare solo le valvole 
Non ho intenzione di usare HA, mi basta un bridge che lavora in locale e metta in comunicazione valvole e "cronotermostato" per pilotare caldaia 
Guarderò funzioni e costi di netatmo e tado come consigliato da voi sopra

**Comment by NuccioT** [comment]
Ho risolto con un aiutante. Ho creato un sensore che in base alla temperatura richiesta in ogni stanza 
indica se la caldaia deve essere accesa o spenta. In base allo stato lo switch sulla caldaia da o nega il consenso per l'accensione. Il problema con le valvole smart è che registrano le temperature vicino al radiatore e quindi i valori sono poco attendibili. Bisognerebbe mettere un termometro in ogni stanza. Inoltre con questo sistema il comfort è garantito ma i consumi ne risentono. Personalmente ho risolto con un programmatore orario. Nelle stanze che non serve la temperatura è regolata sui 15 gradi con un giorno di accensione a settimana per stemperare il clima.
```
{% if
(states('sensor.giorno_numero')|float > 105) and (states('sensor.giorno_numero')|float < 280) and
(state_attr('climate.z2m_r103_radiatore_cucina', 'current_temperature') >  state_attr('climate.z2m_r103_radiatore_cucina', 'temperature')) and 
(state_attr('climate.z2m_r107_radiatore_bagnetto', 'current_temperature') >  state_attr('climate.z2m_r107_radiatore_bagnetto', 'temperature')) and
(state_attr('climate.z2m_gruppo_sala_tv', 'current_temperature') >  state_attr('climate.z2m_gruppo_sala_tv', 'temperature')) and 
(state_attr('climate.z2m_r110_radiatore_ingresso_nord', 'current_temperature') >  state_attr('climate.z2m_r110_radiatore_ingresso_nord', 'temperature')) and
(state_attr('climate.z2m_r201_radiatore_bagno_xxx', 'current_temperature') >  state_attr('climate.z2m_r201_radiatore_bagno_xxx', 'temperature')) and 
(state_attr('climate.z2m_r202_radiatore_camera_xxx', 'current_temperature') >  state_attr('climate.z2m_r202_radiatore_camera_xxx', 'temperature')) and
(state_attr('climate.z2m_gruppo_scala_ovest', 'current_temperature') >  state_attr('climate.z2m_gruppo_scala_ovest', 'temperature')) and 
(state_attr('climate.z2m_r204_radiatore_corridoio', 'current_temperature') >  state_attr('climate.z2m_r204_radiatore_corridoio', 'temperature')) and 
(state_attr('climate.z2m_r205_radiatore_camera_xxx', 'current_temperature') >  state_attr('climate.z2m_r205_radiatore_camera_xxx', 'temperature')) and
(state_attr('climate.z2m_r206_radiatore_bagno', 'current_temperature') >  state_attr('climate.z2m_r206_radiatore_bagno', 'temperature')) and 
(state_attr('climate.z2m_r207_radiatore_cameretta', 'current_temperature') > state_attr('climate.z2m_r207_radiatore_cameretta', 'temperature'))
%} Off
{% else %} On
{% endif %}
```

**Reply by trebestie** [reply]
Io ho messo un termometro in ogni stanza e impostato varie opzioni su cui regolare la temperatura: media globale, solo uno, media di due...
La caldaia è comandata da uno Shelly anche se ho lasciato un termostato a cursori di backup.
Le valvole termostatiche sono generalmente tutte aperte tranne alcuni casi in cui mi servono per particolari regolazioni fini.
Il confort è notevole, il consumo non saprei, spendo una fortuna da sempre
